File: 2076-sum-of-digits-of-string-after-convert/2076-sum-of-digits-of-string-after-convert.cpp
class Solution {
public:
    int getLucky(string s, int k) {
        string worker = "";
        for (auto i: s) {
            worker += to_string(i - 'a' + 1);
        }
        int m = 0;
        while (k) {
            m = 0;
            for (auto i: worker) {
                m += i - '0';
            }
            worker = to_string(m);
            k--;
        }
        return m;
    }
};

File: 2076-sum-of-digits-of-string-after-convert/2076-sum-of-digits-of-string-after-convert.py
class Solution:
    def getLucky(self, s: str, k: int) -> int:
        m = ""
        for i in s:
            m += str(ord(i) - ord('a') + 1)

        ans = 0
        while k > 0:
            ans = 0
            for digit in m:
                ans += int(digit)
            k -= 1
            m = str(ans)
        return ans
